swift_browser_ui_frontend/src/components/ObjectTable.vue

```diff
@@ -63,7 +63,7 @@
             outlined
             icon-left="upload"
             tag="router-link"
-            :to="{name: 'UploadView', params: {
+            :to="{name: 'UploadModal', params: {
```

Comment:
This shouldn't be a router link to `UploadModal`, which ends up just leading to the front page as this route doesn't exist.
`@click="toggleUploadModal"` works here as well.